src/platform/Linux/CHIPBluezHelper.h

```diff
@@ -205,6 +205,58 @@ CHIP_ERROR StopDiscovery(BluezEndpoint * apEndpoint);
 
 CHIP_ERROR ConnectDevice(BluezDevice1 * apDevice);
 
+/// Iterates over available BlueZ adapters
+///
+/// Usage example:
+///
+///  AdapterIterator iterator;
+///  while (iteerator.Next()) {
+///      std::cout << iterator.GetAddress() << std::endl;
+///  }
+class AdapterIterator
+{
+public:
+    ~AdapterIterator();
+
+    /// Moves to the next DBUS interface.
+    ///
+    /// MUST be called at least once
```

Comment:
Could the destructor check if `mManager != nullptr` instead of setting this requirement?

Reply:
Comment was akward,  meant that it needs to be called before current value has any meaning. Updated comments a bit, hoping it is clearer now.